plugins/linear-genome-view/src/LinearGenomeView/model.ts

```diff
@@ -243,6 +243,12 @@ export function stateModelFactory(pluginManager: PluginManager) {
          * show the "gridlines" in the track area
          */
         showGridlines: true,
+
+        /**
+         * #property
+         * highlights on the LGV from the URL parameters
+         */
+        highlight: types.optional(types.frozen(), ''),
```

Comment:
the default being a string may be misleading as it suggests it is a string in some cases. if it is not ever a string and actually has a certain format, you can use typescript for the types.frozen, or, even make it an actual types.model

see https://mobx-state-tree.js.org/overview/types for the typescripting of types.frozen